common/components/common/event_projects/AboutProjectEventDisplay.jsx

```diff
@@ -0,0 +1,439 @@
+            <div className="ProjectCard-logo">
+              {eventProject?.event_thumbnail?.publicUrl ? (
+                <img
+                  src={eventProject.event_thumbnail.publicUrl}
+                  alt="Event Logo"
+                ></img>
+              ) : null}
+            </div>
+          </div>
+        </div>
+        <div className="Profile-top-section-content">
+          {showVideo && (
+            <VideoModal
+              showModal={this.state.showVideoModal}
+              onClose={() => {
+                this.setState({ showVideoModal: false });
+              }}
+              videoUrl={this.state.videoLink.linkUrl}
+              videoTitle={eventProject.project_name}
+            />
+          )}
+          <div
+            className="Profile-top-logo"
+            onClick={() => {
+              showVideo && this.setState({ showVideoModal: true });
+            }}
+          >
+            {eventProject?.project_thumbnail?.publicUrl ? (
+              <div>
+                {showVideo && (
+                  <div className="ProjectCard-play-button">
+                    <PlaySVG />
+                  </div>
+                )}
+                <img
+                  src={eventProject.project_thumbnail.publicUrl}
+                  alt="Project Logo"
+                />
+              </div>
+            ) : null}
+          </div>
+          <div className="Profile-top-details">{this._renderIconList()}</div>
+          <div className="Profile-top-interactions">
+            {showEdit && (
+              <Button variant="primary" href={editUrl}>
+                Edit
+              </Button>
+            )}
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  // primary section content
+  _renderPrimarySection(eventProject: EventProjectAPIDetails): React$Node {
+    const comingSoonMsg: React$Node = (
+      <React.Fragment>Coming soon!</React.Fragment>
+    );
+
+    return (
+      <div className="Profile-primary-container">
+        <div className="tab-content AboutProjectEvent-primary-container">
+          <h3>About</h3>
+          <p>{eventProject?.project_short_description}</p>
+
+          <h3>Problem</h3>
+          <p>{eventProject?.project_description}</p>
+
+          {eventProject?.project_description_solution && (
+            <React.Fragment>
+              <h3>Solution</h3>
+              <p>{eventProject.project_description_solution}</p>
+            </React.Fragment>
+          )}
+
+          <h3>Hackathon Goal</h3>
+          {eventProject?.event_project_goal ? (
+            <p>{eventProject.event_project_goal}</p>
+          ) : (
+            comingSoonMsg
+          )}
+
+          <h3>Planned Scope</h3>
+          {eventProject?.event_project_scope ? (
+            <p>{eventProject.event_project_scope}</p>
+          ) : (
+            comingSoonMsg
+          )}
+
+          {/*TODO: Show newlines*/}
```

Comment:
todo, but I think this one's done? If I remember the tests we ran before deploy....?